### PR TITLE
[stable/oauth2-proxy] New release security fix + organisation renaming

### PR DESCRIPTION
**Pull request by pollier**
Fix a security issue
And migrate to the new organisation / naming
https://github.com/oauth2-proxy/oauth2-proxy/releases/tag/v5.1.1

@desaintmartin 

@Miouge1 

#### What this PR does / why we need it:
New release fix a security issue
Upstream organisation / name changed

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/oauth2-proxy/README.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ Due to [this issue](https://github.com/helm/helm/issues/6583) there may be error
 
 Version 3.0.0 introduces support for [EKS IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) by adding a managed service account to the chart.  This is a breaking change since the service account is enabled by default.  To disable this behaviour set `serviceAccount.enabled` to `false`
 
+### To 3.0.1
```
Comment (desaintmartin) [inline stable/oauth2-proxy/README.md:54]: No need for patch / minor release. Convention is to use proper PR name (which is used as commit message).

**Comment by desaintmartin** [comment]
There is a conflict, could you rebase?

**Comment by pollier** [comment]
> There is a conflict, could you rebase?

Done

**Comment by abdennour** [comment]
Any update on this PR ?
I think it is related to #21198

**Comment by abdennour** [comment]
@desaintmartin  Could you please give this PR a look ?
The owner of PR updated since 5 days

**Comment by mbarrien** [comment]
Overlaps with #22368, but #22368 made more of the naming changes.

**Pull request closed without merging**